tests/Microsoft.Identity.Web.Test/SignedAssertionFilePathCredentialsLoader.cs

```diff
@@ -3,81 +3,76 @@
-
+        private const string FilePath = "signedAssertion.txt";
+        private const string FilePath2 = "signedAssertion2.txt";
+        private const string AksEnvironmentVariableName = "AZURE_FEDERATED_TOKEN_FILE";
+        private readonly string _token;
+        private readonly SignedAssertionFilePathCredentialsLoader _signedAssertionFilePathCredentialsLoader = new(null);
 
         public SignedAssertionFilePathCredentialsLoaderTests()
         {
-            JsonWebTokenHandler handler = new JsonWebTokenHandler();
-            token = handler.CreateToken("{}");
+            JsonWebTokenHandler handler = new();
+            _token = handler.CreateToken("{}");
         }
 
-        [Fact]
-        public async Task GetClientAssertion_WhenSpecifiedSignedAssertionFileExists_ReturnsClientAssertion()
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task GetClientAssertion_WhenSpecifiedSignedAssertionFileExists_ReturnsClientAssertion(bool withEnvVariable)
         {
             // Arrange
-            File.WriteAllText(filePath, token.ToString());
-            CredentialDescription credentialDescription = new CredentialDescription
+            var filePath = withEnvVariable ? FilePath : FilePath2;
+            File.WriteAllText(filePath, _token.ToString());
+            CredentialDescription credentialDescription;
+            if (withEnvVariable)
             {
-                SourceType = CredentialSource.SignedAssertionFilePath,
-                SignedAssertionFileDiskPath = filePath
-            };
+                Environment.SetEnvironmentVariable(AksEnvironmentVariableName, filePath);
+                credentialDescription = new()
+                {
+                    SourceType = CredentialSource.SignedAssertionFilePath,
+                };
+            }
+            else
+            {
+                credentialDescription = new()
+                {
+                    SourceType = CredentialSource.SignedAssertionFilePath,
+                    SignedAssertionFileDiskPath = filePath
+                };
+            }
 
             // Act
-            await signedAssertionFilePathCredentialsLoader.LoadIfNeededAsync(credentialDescription, null);
+            await _signedAssertionFilePathCredentialsLoader.LoadIfNeededAsync(credentialDescription, null);
 
             // Assert
             Assert.NotNull(credentialDescription.CachedValue);
 
             // Delete the signed assertion file.
             File.Delete(filePath);
-        }
-
-        [Fact]
-        public async Task GetClientAssertion_WhenEnvironmentVariablePointsToSignedAssertionFileExists_ReturnsClientAssertion()
-        {
-            // Arrange
-            File.WriteAllText(filePath, token.ToString());
-            Environment.SetEnvironmentVariable(aksEnvironmentVariableName, filePath);
-            CredentialDescription credentialDescription = new CredentialDescription
+            if (withEnvVariable)
```

Comment:
does it build? It seem that this is code outside of a method? (from what the diff provides in GitHub. Might be a tooling issue?

Reply:
I think it's a tooling issue as i've been running the tests locally.